src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const obj = {};
+  const str = sourceString;
+  const stringWithNoBreakLInes = str.replace(/\n*/g, '');
+  const strArr = stringWithNoBreakLInes.split(';');
+
+  for (let i = 0; i < strArr.length; i++) {
+    if (strArr[i].includes(':')) {
+      const keyValue = strArr[i].split(':');
+
+      const forKey = keyValue[0].trim();
+      const forValue = keyValue[1].trim();
```

Comment:
you can desctructure those from keyValue
```suggestion
      const [key, value] = keyValue;
```